common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsManager.kt

```diff
@@ -0,0 +1,101 @@
+package com.revenuecat.purchases.common.diagnostics
+
+import android.content.Context
+import android.content.SharedPreferences
+import androidx.annotation.VisibleForTesting
+import com.revenuecat.purchases.common.Backend
+import com.revenuecat.purchases.common.Dispatcher
+import com.revenuecat.purchases.common.verboseLog
+import java.io.IOException
+
+class DiagnosticsManager(
```

Comment:
Not sure what other types in the Android codebase do, but it would help to have a quick docstring here summarizing what this type does.

Reply:
We don't usually add docstrings from what I've seen. But yeah, I can add this here 👍